src/service_client/logger.py

```diff
@@ -32,28 +32,72 @@ def format(self, record):
         return self._filter(original)
 
 
+class ColorizingStreamHandler(logging.StreamHandler):
+    BLUE = "\033[0;34m"
+    LIGHT_RED = "\033[1;31m"
+    LIGHT_YELLOW = "\033[1;33m"
+    LIGHT_BLUE = "\033[1;34m"
+    LIGHT_PURPLE = "\033[1;35m"
+    LIGHT_CYAN = "\033[1;36m"
+    WHITE = "\033[1;37m"
+    RESET = "\033[0m"
+
+    def __init__(self, *args, **kwargs):
+        self._colors = {
+            logging.DEBUG: self.BLUE,
+            logging.INFO: self.RESET,
+            logging.WARNING: self.LIGHT_YELLOW,
+            logging.ERROR: self.LIGHT_RED,
+            logging.CRITICAL: self.LIGHT_PURPLE,
+        }
+        super(ColorizingStreamHandler, self).__init__(*args, **kwargs)
```

Comment:
To allow future-compatibility for when they'll disallow ``super`` with arguments:
```suggestion
        super().__init__(*args, **kwargs)
```